src/ components/Tile.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Tile({img, imgTitle, title, children}) {
+    return (
+        <section>
+        <img src={img} alt={imgTitle}/>
```

Comment:
Voor de gebruiksvriendelijkheid van E-readers is het beter om {img && <img src={img} alt={imgDescription}/>} te gebruiken, dan wordt het element niet ingeladen als er geen img is.